File: ws/codena/u2/ws13ArraysJson/Python/view/product_view.py
class ProductView:

    def show_menu(self):

        print("\n===== STORE SYSTEM =====")

        print("1. Add products and save to JSON")
        print("2. Read JSON and show weights in kilograms")
        print("3. Exit")

    def show_message(self, message):

        print(message)

    def show_products(self, products):

        print("\n=== PRODUCTS IN KILOGRAMS ===")

        for product in products:

            print(
                f"Name: {product['name']} | "
                f"Kilograms: "
                f"{product['weight_kilograms']:.2f} kg"
            )

File: ws/codena/u2/ws13ArraysJson/JavaScript/view/ProductView.js
class ProductView {

    showMenu() {

        console.log("\n===== STORE SYSTEM =====");

        console.log("1. Add products and save to JSON");
        console.log("2. Read JSON and show weights in kilograms");
        console.log("3. Exit");
    }

    showMessage(message) {

        console.log(message);
    }

    showProducts(products) {

        console.log("\n=== PRODUCTS IN KILOGRAMS ===");

        products.forEach(product => {

            console.log(
                `Name: ${product.name} | Kilograms: ${product.weightKilograms.toFixed(2)} kg`
            );
        });
    }
}

module.exports = ProductView;